api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -523,11 +534,14 @@ ml_single_close (ml_single_h single)
     return ML_ERROR_INVALID_PARAMETER;
   }
 
+  G_LOCK (magic);
   single_h = (ml_single *) single;
   ML_SINGLE_MAGIC_CHECK (single_h);
```

Comment:
Wouldn't be "safer" to have the macro defines:
```
  G_LOCK (magic);
  single_h = (ml_single *) single;	  
  ML_SINGLE_MAGIC_CHECK (single_h);
```
in a single macro? e.g, ```#define ML_SINGLE_GET_VALID_HANDLE```.

Reply:
Updated